string object_name
---
geometry_msgs/Pose[] holes_poses